### ISSUE TITLE
Viewable impression tracker not firing

### ISSUE THREAD

**Issue by DarkCC7** [radiantmediaplayer/rmp-vast#33]
Hi,
It seems that the viewable impression tracker is not firing. I tried with this VAST 4 sample:
https://github.com/InteractiveAdvertisingBureau/VAST_Samples/blob/master/VAST%204.0%20Samples/Viewable_Impression-test.xml
Best

**Comment by radiantmediaplayer** [comment]
Yes we know about that. The issue is that the ViewableImpression tag is in a Wrapper and not in an Inline element. If you try view like at https://www.radiantmediaplayer.com/vast/tags/inline-linear-viewable.xml then is should work. I have opened a ticket at https://github.com/dailymotion/vast-client-js/issues/378 for this now.

**Comment by radiantmediaplayer** [comment]
This is now included with rmp-vast 6.0.0 (through upgrade to [vast-client-js](https://github.com/dailymotion/vast-client-js) 4.0.0)

**Comment by edihasaj** [comment]
Even with InLine tag is not working on version 4 of vast? is there any way I can get it firing?

```
<VAST version="4.0" xmlns="http://www.iab.com/VAST">
  <Ad id="20004" sequence="1" conditionalAd="false">
    <InLine>
      <AdSystem version="4.0">iabtechlab</AdSystem>
      <Error>http://example.com/error</Error>
      <Extensions>
        <Extension type="iab-Count">
          <total_available><![CDATA[ 2 ]]></total_available>
        </Extension>
      </Extensions>
      <Impression id="Impression-ID">http://example.com/track/impression2</Impression>
      <ViewableImpression id="1543">
        <Viewable>
          <![CDATA[http://example.com/track/impression]]>
        </Viewable>
        <NotViewable>
          <![CDATA[http://search.iabtechlab.com/error?errcode=102&imprid=s5-ea2f7f298e28c0c98374491aec3dfeb1&ts=1243]]>
        </NotViewable>
        <ViewUndetermined>
          <![CDATA[http://search.iabtechlab.com/error?errcode=102&imprid=s5-ea2f7f298e28c0c98374491aec3dfeb1&ts=1243]]>
        </ViewUndetermined>
      </ViewableImpression>
      <Pricing model="cpm" currency="USD"><![CDATA[ 25.00 ]]></Pricing>
      <AdTitle><![CDATA[VAST 4.0 Pilot - Scenario 5]]></AdTitle>
      <Creatives>
        <Creative id="5480" sequence="1" adId="2447226">
          <CompanionAds>
            <Companion id="1232" width="100" height="150" assetWidth="250" assetHeight="200" expandedWidth="350" expandedHeight="250" adSlotID="3214" pxratio="1400">
              <StaticResource creativeType="image/png"><![CDATA[https://www.iab.com/wp-content/uploads/2014/09/iab-tech-lab-6-644x290.png]]></StaticResource>
              <CompanionClickThrough><![CDATA[https://iabtechlab.com]]></CompanionClickThrough>
            </Companion>
          </CompanionAds>
          <UniversalAdId idRegistry="Ad-ID" idValue="8465">8465</UniversalAdId>
        </Creative>
        <Creative id="5480" sequence="1" adId="2447226">
          <Linear>
            <TrackingEvents>
              <Tracking event="start">http://example.com/tracking/start</Tracking>
              <Tracking event="firstQuartile">http://example.com/tracking/firstQuartile</Tracking>
              <Tracking event="midpoint">http://example.com/tracking/midpoint</Tracking>
              <Tracking event="thirdQuartile">http://example.com/tracking/thirdQuartile</Tracking>
              <Tracking event="complete">http://example.com/tracking/complete</Tracking>
              <Tracking event="progress" offset="00:00:10">http://example.com/tracking/progress-10</Tracking>
            </TrackingEvents>
            <Duration>00:00:16</Duration>
            <MediaFiles>
              <MediaFile id="5241" delivery="progressive" type="video/mp4" bitrate="2000" width="1280" height="720" minBitrate="1500" maxBitrate="2500" scalable="1" maintainAspectRatio="1" codec="H.264"><![CDATA[https://iab-publicfiles.s3.amazonaws.com/vast/VAST-4.0-Short-Intro.mp4]]></MediaFile>
              <MediaFile id="5244" delivery="progressive" type="video/mp4" bitrate="1000" width="854" height="480" minBitrate="700" maxBitrate="1500" scalable="1" maintainAspectRatio="1" codec="H.264"><![CDATA[https://iab-publicfiles.s3.amazonaws.com/vast/VAST-4.0-Short-Intro-mid-resolution.mp4]]></MediaFile>
              <MediaFile id="5246" delivery="progressive" type="video/mp4" bitrate="600" width="640" height="360" minBitrate="500" maxBitrate="700" scalable="1" maintainAspectRatio="1" codec="H.264"><![CDATA[https://iab-publicfiles.s3.amazonaws.com/vast/VAST-4.0-Short-Intro-low-resolution.mp4]]></MediaFile>
            </MediaFiles>
            <VideoClicks>
              <ClickThrough id="blog"><![CDATA[https://iabtechlab.com]]></ClickThrough>
            </VideoClicks>
          </Linear>
          <UniversalAdId idRegistry="Ad-ID" idValue="8465">8465</UniversalAdId>
        </Creative>
      </Creatives>
      <Description><![CDATA[This is sample companion ad tag with Linear ad tag. This tag while showing video ad on the player, will show a companion ad beside the player where it can be fitted. At most 3 companion ads can be placed. Modify accordingly to see your own content. ]]></Description>
    </InLine>
  </Ad>
</VAST>
```

I took this example somewhere.

Any help would be much appreciated
